Sources/ZcashLightClientKit/Account/Account.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Account.swift
+//  ZcashLightClientKit
+//
+//  Created by Lukáš Korba on 2024-11-19.
+//
+
+/// A [ZIP 32](https://zips.z.cash/zip-0032) account index.
+public struct Zip32Account: Equatable, Codable, Hashable {
```

Comment:
I think we should make it clearer that this is a ZIP 32 account *index*. That is, ...
```suggestion
/// A [ZIP 32](https://zips.z.cash/zip-0032) account *index*.
///
/// Note that this must be paired with a seed (or other context that
/// specifies a seed) in order to identify a ZIP 32 *account*.
public struct Zip32AccountIndex: Equatable, Codable, Hashable {
```

Reply:
Updated